### DISCUSSION TITLE
LTMeat Flood still works surprisingly

### DISCUSSION THREAD

**Discussion by DankFireD** [3kh0/ext-remover#1162] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Not really a fix, persay, but, this is just a post saying that it still works as I tried it on my school Chromebook and its not caubified so.... it still seems to be on v127 though but its still worth a try on v128

**Comment by wvminecraftkid** [comment]
I'm using ltmeat flood and it sometimes works but lots of time it only stays for a few minutes before getting blocked again.

My process

Open the window (I'm using 740 tabs because any more and my Chromebook crashes.)
Close the window as soon as it opens (is this correct or should I wait for all the tabs to load?)
Immediatly open the window again
As fast as possible flip allow access to file urls
Close the window
This is pretyy unrealiable so I think I'm doing something wrong. What should I improve? Sometimes it ends up working and lasts for around 48 hours but usally not

**Reply by DankFireD** [reply]
i just do it again after the initial time it comes back, our school's chromebooks are kept in the classroom and we're not responsible for our own and it lasts for the rest of the period for me sooo

**Comment by wvminecraftkid** [comment]
Do i just flick the switch once or spam it a bit?

**Reply by DankFireD** [reply]
i just did it once

**Comment by AshtonDavies** [comment]
Suddenly everyone's using this exploit again, I get DMs about it and the kajig releases...

**Reply by DankFireD** [reply]
I mean its simple and easy to do so I can't blame 'em

**Comment by schoolexploitkid** [comment]
I use it sometimes, although I spam the allow access to file urls switch
